diff --git a/node_modules/@parcel/core/lib/CommittedAsset.js b/node_modules/@parcel/core/lib/CommittedAsset.js
index 398eeb1..b25deb3 100644
--- a/node_modules/@parcel/core/lib/CommittedAsset.js
+++ b/node_modules/@parcel/core/lib/CommittedAsset.js
@@ -9,6 +9,8 @@ var _v = _interopRequireDefault(require("v8"));
 
 var _stream = require("stream");
 
+const { readFileSync } = require("fs");
+
 var _sourceMap = _interopRequireDefault(require("@parcel/source-map"));
 
 var _utils = require("@parcel/utils");
@@ -56,7 +58,8 @@ class CommittedAsset {
           return content;
         }));
       } else {
-        throw new Error('Asset has no content');
+        // throw new Error('Asset has no content');
+        return readFileSync(this.value.filePath);
       }
     }
 
diff --git a/node_modules/@parcel/core/lib/test.tsxt b/node_modules/@parcel/core/lib/test.tsxt
new file mode 100644
index 0000000..396a0ba
--- /dev/null
+++ b/node_modules/@parcel/core/lib/test.tsxt
@@ -0,0 +1 @@
+yes
\ No newline at end of file
